odoo/addons/base/wizard/base_language_install.py

```diff
@@ -28,12 +28,16 @@ def _get_languages(self):
                                help="If you check this box, your customized translations will be overwritten and replaced by the official ones.")
     state = fields.Selection([('init', 'init'), ('done', 'done')],
                              string='Status', readonly=True, default='init')
+    remote_translations = fields.Boolean("Fetch Online Translations", default=True,
```

Comment:
@mart-e wouldn't it be better to set this to `False` by default? So that you have to turn it on yourself, by purpose. I can imagine people going nuts about changing translations if they don't know about this 😉

Reply:
@Yenthe666 please, see my comment on top of the PR, this is still at the very early stage, not even sure it will be accepted ;-)